Jenkinsfile

```diff
@@ -0,0 +1,53 @@
+@Library('shared-libraries') _
+pipeline{
+  agent {label 'javaClientLinuxPool'}
+  options {
+    checkoutToSubdirectory 'java-client-api'
+    buildDiscarder logRotator(artifactDaysToKeepStr: '7', artifactNumToKeepStr: '', daysToKeepStr: '30', numToKeepStr: '')
+  }
+  environment{
+    JAVA_HOME_DIR="/home/builder/java/openjdk-1.8.0-262"
+    GRADLE_DIR   =".gradle"
+    DMC_USER     = credentials('MLBUILD_USER')
+    DMC_PASSWORD = credentials('MLBUILD_PASSWORD')
+  }
+  stages{
+    stage('tests'){
+      steps{
+        copyRPM 'Release','10.0-9.4'
```

Comment:
@anu3990  @llinggit  Do you think we should use the latest ML release or the latest nightly build?

Reply:
sorry I missed the context here. We've already got a Jenkins job to run java unit tests against nightly build, right?

Reply:
This will run against submitted PRs before they're merged

Reply:
We can also run the tests on a nightly server after merging the PR to develop.

Reply:
nightly build is better I think.